framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -25,6 +25,8 @@
 #include "audio/audio_manager.h"
 
 namespace media {
+//ToDo: This handler needs to be moved into MediaWorker.cpp
+audio_handler_t* mOutAudioHandler;
```

Comment:
I feel coding guidelines violated,
audio_handler_t* mOutAudioHandler;
==>(change to)
audio_handler_t *mOutAudioHandler;